### PR TITLE
Gameday2 match ticker

### PR DESCRIPTION
**Pull request by fangeugene**
image

### CODE REVIEW

File: react/gameday2/components/ChatSidebar.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ class ChatSidebar extends React.Component {
     const currentChat = this.props.chats[this.props.currentChat]
     let currentChatName = 'UNKNOWN'
     if (currentChat) {
-      currentChatName = currentChat.name
+      currentChatName = `${currentChat.name} Chat`
```
Comment (nwalters512) [inline react/gameday2/components/ChatSidebar.js]: Good call :+1:

File: react/gameday2/components/VideoCellToolbar.js [hunk 2]
```diff
@@ -17,6 +18,47 @@ const VideoCellToolbar = (props) => {
     fontSize: 16,
   }
 
+  const matchTickerGroupStyle = {
+    flexGrow: 1,
+    width: '0%',  // Slightly hacky. Prevents ticker from bleeding into next cell
```
Comment (nwalters512) [inline react/gameday2/components/VideoCellToolbar.js]: I'll trust that this works :stuck_out_tongue:

**Comment by nwalters512** [comment]
Looks good to me! You're a natural at React.

**Comment by fangeugene** [comment]
I just copied your patterns 😛

**Pull request merged**